pkg/maps/remotelxcmap/remotelxcmap.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2016-2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package remotelxcmap
```

Comment:
I'm not too fond of the name remotelxcmap. This map should really not be related to lxc or endpoints at all. It maps ips to identities. Cann we just call it `ipcache`?